docs/docs/building.md

```diff
@@ -27,89 +26,63 @@ contract MyContract is Migratable {
 }
 ```
 
-You might have noted that our sample contract has an `initialize` function instead of the standard contstructor. This is a requirement of ZeppelinOS's upgradeabiltiy system, you can learn more about this in our [Advanced topics](advanced.md) section. 
+Notice that our sample contract has an `initialize` function: this works as a constructor for our upgradeable contract. This is a requirement of the [ZeppelinOS's upgradeability system](advanced.md#initializers-vs-constructors).
```

Comment:
Repeats: our sample contract, our upgradeable contract. Remove "the". ->

Notice that our sample contract has an `initialize` function instead of the standard constructor. This is a requirement of [ZeppelinOS's upgradeability system](advanced.md#initializers-vs-constructors).